mangaki/mangaki/utils/recommendations.py

```diff
@@ -5,119 +5,142 @@
 from django.contrib.auth.models import User
 from django.db.models import Count
 from django.db import connection
+import random
+from datetime import datetime
+from mangaki.utils.wals import MangakiWALS
+from mangaki.utils.als import MangakiALS
+from mangaki.utils.knn import MangakiKNN
+from mangaki.utils.svd import MangakiSVD
+import numpy as np
+import pickle
+import os.path
+
+
+NB_NEIGHBORS = 20
+NB_RECO = 10
+RATED_BY_AT_LEAST = 2
+CHRONO_ENABLED = False
```

Comment:
This file should probably be refactored into a management command and those should be parameters of that command, I think (let's do it in a later patch).